test/functional/test_framework/test_node.py

```diff
@@ -5,6 +5,7 @@
 """Class for bitcoind node under test"""
 
 import contextlib
+from collections import Counter
```

Comment:
(Could possibly use `collections.defaultdict(int)` instead of `Counter` iff I really need to retouch).